packages/askui-nodejs/src/execution/ui-control-client.ts

```diff
@@ -130,6 +130,24 @@ export class UiControlClient extends FluentCommand {
     }
   }
 
+  /**
+   * Waits for `<delayInMs>` ms, e.g., 1000 ms. The exact delay may be a little longer
```

Comment:
only question regarding,  The exact delay may be a little longer why is it the case

Reply:
Because you can never be a 100% sure when the event loop is going to pick this up again, only that it will be after > 1000 ms (see https://stackoverflow.com/questions/49811043/relationship-between-event-loop-libuv-and-v8-engine)